### DISCUSSION TITLE
embed stream in simple webpage?

### DISCUSSION THREAD

**Discussion by mrhhats** [GerryDazoo/Slinger#345] [General]
Hey everyone! anyone taking the VLC stream URL and putting it into a simple HTML page?

**Comment by GerryDazoo** [comment]
Yes. didn't work for me. But I may have done something wrong.

**Reply by mrhhats** [reply]
I wonder if its due to needing a possible .mp4 extension on the end of the URL? some example code I tried seem to have that but the slinger URL didn't work for me. Thanks Gerry for all you did!

**Reply by GerryDazoo** [reply]
It's not an MP4 video steam

**Comment by sp222222** [comment]
> Hey everyone! anyone taking the VLC stream URL and putting it into a simple HTML page?

this would be nice then I could get to all mine from my webserver :) I wonder what else it needs to complete the stream/link ?

**Comment by GerryDazoo** [comment]
You would need to setup a proxy with something like ffmpeg to re-encode the stream from Slinger into something a web browser could display.

**Reply by mrhhats** [reply]
I wonder how possible it would be pipe it into a simple OwnCast server?
https://owncast.online/

I'm going to see if i can install OBS and somehow grab the stream them publish to RTMP/RTSP
